src/components/themed/EdgeText.js

```diff
@@ -16,12 +16,16 @@ class EdgeTextComponent extends React.PureComponent<OwnProps & ThemeProps> {
   render() {
     const { children, style, theme, ...props } = this.props
     const { text } = getStyles(theme)
+    let numberOfLines = this.props.numberOfLines || 1
+    if (typeof children === 'string' && children.includes('\n')) {
+      numberOfLines = numberOfLines + (children.match(/\n/g) || []).length
+    }
     return Platform.OS === 'ios' ? (
-      <Text style={[text, style]} numberOfLines={this.props.numberOfLines || 1} adjustsFontSizeToFit {...props}>
+      <Text style={[text, style]} numberOfLines={numberOfLines} adjustsFontSizeToFit {...props}>
         {children}
       </Text>
     ) : (
-      <Text style={[text, style]} numberOfLines={this.props.numberOfLines || 1} ellipsizeMode={this.props.ellipsizeMode || 'middle'} {...props}>
+      <Text style={[text, style]} numberOfLines={numberOfLines} ellipsizeMode={this.props.ellipsizeMode || 'middle'} {...props}>
```

Comment:
adjustsFontSizeToFit Now works on android in rn63. Please enable

Reply:
Created a new commit for that fix. Fixed